docs/core/whats-new/dotnet-core-3-0.md

```diff
@@ -278,6 +326,10 @@ Starting with Preview 3, running .NET Core 3.0 on Linux with Docker works better
 * Explicit size can be set as an absolute number or percentage of cgroup limit.
 * Minimum reserved segment size per GC heap is 16 mb. This size reduces the number of heaps that are created on machines.
 
+## Smaller Garbage Collection heap sizes
+
+While the default Garbage Collector (GC) heap size was reduced for Docker, memory usage has been improved while not running in a container. This better aligns with the generation 0 allocation budget with modern processor cache sizes.
```

Comment:
The connection between the while clause and the main part of the sentence is unclear.

Reply:
Ahh yes.. A bad merge between two different sentences. I'll fix that.